src/discord/commands/moderation/kick.ts

```diff
@@ -23,13 +17,17 @@ export const KickCommand: Command = {
                     type: "string",
                     required: false,
                     unlimited: true
-                }]
-            )
+                }
+            ])
         ]
     },
     handler: async (msg, next) => {
         const [user, reason] = msg.args;
 
-        await msg.reply(`And this is the point where I'd usually kick <@${(user as any).id}> for ${reason || "no reason"} but i'm in debug mode.`);
+        await msg.reply(
```

Comment:
Can prettierc be configured to not do this?

Reply:
It's literally not cute

Reply:
Will be fixed next commit.